docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -30,16 +34,60 @@ by default in this version of Determined are described below.
 See :doc:`/training/setup-guide/set-environment-images` for the images Docker Hub location and add
 each tagged image required by your environment and experiments to the image cache.
 
-************
- Add Images
-************
+.. _referencing-local-image-paths:
+
+*******************************
+ Referencing Local Image Paths
+*******************************
+
+Each container runtime supports various local container file formats and references them using a
+slightly different syntax. Utilize a cached image by referencing a local path using the experiment
+configuration :ref:`environment.image <exp-environment-image>`.
+
+When using PodMan, you could pull and then save images in OCI archive format to files in a local
+directory ``/shared/containers`` and then reference using the syntax below.
+
+   .. code:: yaml
+
+      environment:
+         image: oci-archive:/shared/containers/cuda-11.3-pytorch-1.10-tf-2.8-gpu
+
+When using Singularity, you could save SIF files in a local directory ``/shared/containers`` and
+then reference them via full path using the syntax below.
+
+   .. code:: yaml
+
+      environment:
+         image: /shared/containers/cuda-11.3-pytorch-1.10-tf-2.8-gpu.sif
+
+Set these ``image`` file references above as the default by specifying them in the
+:ref:`task_container_defaults <master-task-container-defaults>` section of the
+``/etc/determined/master.yaml`` file.
+
+Note: If you specify an image using :ref:`task_container_defaults <master-task-container-defaults>`,
+you prevent new environment container image versions from being adopted on each update of
```

Comment:
But adopting new images that are hardcoded in each update of Determined is only possible if the compute nodes have access to the Internet, in order to pull them down, correct?

Reply:
Right.   Not necessarily an issue, but using the singularity image cache doesn't have this same restriction.  On an upgrade DAI containers would instead fail to be found instead of sticking with the fixed version.
Just wanted to highlight the differences between the cases so that people could choose.

If you are not using custom images (i.e. only using the default ones), then this is an easier strategy to employ.